PowerEditor/installer/nativeLang/spanish.xml

```diff
@@ -967,8 +971,8 @@ Translation note:
 					<Item id="6134" name="Ocultar"/>
 
 					<Item id="6131" name="Menú"/>
-					<Item id="6122" name="Ocultar barra de opciones (Alt o F10 para activar/desactivar)"/>
-					<Item id="6132" name="Ocultar atajos ＋ ▼ ✕ en la derecha de la barra de menús (es necesario reiniciar Notepad++)"/>
+					<Item id="6122" name="Ocultar barra de opciones (Alt o F10 para mostrarla/ocultarla)"/>
+					<Item id="6132" name="Ocultar atajos ＋ ▼ ✕  a la derecha en barra de menús (reiniciar Notepad++)"/>
```

Comment:
`(reiniciar Notepad++)` should be removed.
And please test your translation to make sure the translated item is fit in the new interface.

Reply:
Sorry, it was translated like this due to lack of space for all the text.

Fixed and added the missing lines. I did the same with the galician.xml